code/modules/reagents/reagent_containers.dm

```diff
@@ -41,11 +41,45 @@
 		label_name = name
 
 	add_initial_reagents()
+	AddElement(/datum/element/liquids_interaction, on_interaction_callback = TYPE_PROC_REF(/obj/item/reagent_containers/glass/beaker, attack_on_liquids_turf))
```

Comment:
calling a proc on the beaker type? this is the parent type, what if it's not a beaker